Composer/packages/client/src/recoilModel/atoms/botState.ts

```diff
@@ -370,6 +371,11 @@ export const isEjectRuntimeExistState = atomFamily<boolean, string>({
   default: false,
 });
 
+export const currentTargetState = atomFamily<PublishTarget, string>({
+  key: getFullyQualifiedKey('currentTarget'),
+  default: {} as PublishTarget,
```

Comment:
why not undefined ?

Reply:
because this state should be an publishTarget type